src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -69,9 +70,10 @@ public static void deleteAllTokens(User user) {
     httpDelete.setHeader(X_WIKIA_INTERNAL_REQUEST, "0");
 
     try (CloseableHttpResponse response = getDefaultClient().execute(httpDelete)) {
-      PageObjectLogging.log("DELETE HEADERS: ", response.toString(), true);
+      PageObjectLogging.log("DELETE TOKENS REQUEST: ", httpDelete.toString(), true);
+      PageObjectLogging.log("DELETE TOKENS RESPONSE: ", response.toString(), true);
     } catch (ClientProtocolException e) {
-      PageObjectLogging.log("CLIENT PROTOCOL EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+      PageObjectLogging.log(CLIENT_PROTOCOL_EXCEPTION, ExceptionUtils.getStackTrace(e), false);
```

Comment:
I believe we should add method PageObjectLogging with `.failure(Throwable throwable)` signature which would be an equivalent to `log(e.getClass().getSimpleName(), ExceptionUtils.getStackTrace(e), false);`

Reply:
https://wikia-inc.atlassian.net/browse/IRIS-4897